tsconfig.base.json

```diff
@@ -32,21 +32,15 @@
       "@island.is/api/domains/aircraft-registry": [
         "libs/api/domains/aircraft-registry/src/index.ts"
       ],
-      "@island.is/api/domains/aosh": [
-        "libs/api/domains/aosh/src/index.ts"
-      ],
+      "@island.is/api/domains/aosh": ["libs/api/domains/aosh/src/index.ts"],
```

Comment:
Curious if this is incorrect reformatting or fixing a formatting mistake from others.

Can you check if your editor is using prettier correctly? Try run `yarn format`?